samples: wifi: shutdown: Clarify meaning of shutdown (#13113)
nrfconnect/sdk-nrf: doc/nrf/device_guides/working_with_nrf/nrf70/developing/powersave.rst

  In this state, the device consumes very low power (~2 µA) and does not retain any state information (apart from the values in the OTP memory).
  The device will only respond to a BUCKEN assertion to wake from the Shutdown state.

> The nRF70 Series transition to and from Shutdown state is automatically managed by the Wi-Fi driver. When the network interface is brought down, the Wi-Fi driver puts
> the nRF70 Series device in Shutdown power state. When the network interface is brought up, the Wi-Fi driver puts the nRF70 Series device in Active state.

Reviewer: ```suggestion
The nRF70 Series transition to and from the Shutdown state is automatically managed by the Wi-Fi driver.
When the network interface is brought down, the driver puts the nRF70 Series device in Shutdown power state.
When the network interface is brought up, the driver puts the nRF70 Series device in Active state.
```

> .. note:: The application needs to re-initialize the Wi-Fi association after the network interface is brought up.

Reviewer: ```suggestion
.. note::
   The application needs to reinitialize the Wi-Fi association after the network interface is brought up.
```

  In this state, the device consumes very low power (~2 µA) and does not retain any state information (apart from the values in the OTP memory).
  The device will only respond to a BUCKEN assertion to wake from the Shutdown state.

> The nRF70 Series transition to and from Shutdown state is automatically managed by the Wi-Fi driver.

Reviewer: ```suggestion
The nRF70 Series transition to and from the Shutdown state is automatically managed by the Wi-Fi driver.
```
